.gitignore

```diff
@@ -3,3 +3,4 @@ pkg/*
 *.log
 localgems
 Gemfile.lock
+vendor/*
```

Comment:
something created `vendor/cache`, but now I've removed it I can't get it to happen again. I'll get rid of this.